graphql-api/src/main/resources/stream-registry.graphql

```diff
@@ -89,24 +77,24 @@ type Mutation {
   upsertDomain(
     name: String!
     description: String!
-    tags: [KeyValueInput!]!
+    tags: Tags!
     type: String!
-    configuration: [KeyValueInput!]!
+    configuration: Config!
   ): Boolean!
```

Comment:
I think these should return the outcome Domain / Schema / Stream

Reply:
I think I agree.